### PR TITLE
React Card Separator Kit

### PR DESCRIPTION
**Pull request by jasperfurniss**
Adds the react version of the card_separator that lives inside the Card kit. 
Screen Shot 2019-12-10 at 1 31 44 PM

### CODE REVIEW

File: app/pb_kits/playbook/packs/site_styles/_site-style.scss [hunk 1]
```diff
@@ -77,3 +77,9 @@
     }
   }
 }
+
+.p-4 {
+  padding: 1.5rem;
+}
+
+
```
Comment (jasoncypret) [inline app/pb_kits/playbook/packs/site_styles/_site-style.scss]: These changes aren't necessary

File: app/pb_kits/playbook/pb_card/docs/_card_separator.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        }
+      </Body>
+      <SectionSeparator variant="card" />
+      <Body className="p-4">{"Footer"}</Body>
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_card/docs/_card_separator.jsx]: Remove p-4 till we have a margin/padding solution

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- New React Style - Card Kit ([#496][] @jasperfurniss)
```
Comment (jasoncypret) [inline CHANGELOG.md]: Can you add a more detailed description for Ninjas?

Comment (rafbgarcia) [reply CHANGELOG.md]: Also move to the Add under `[Unreleased]`? :)

**Comment by jasoncypret** [comment]
@jasperfurniss looks good, you just need to rebase

**Pull request merged**